app/ui/src/app/connections/detail-page/info.component.ts

```diff
@@ -1,19 +1,21 @@
-import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { Component, Input, Output, EventEmitter, SimpleChanges } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
 import { Observable } from 'rxjs/Observable';
 import { Subscription } from 'rxjs/Subscription';
 
 import { Connection } from '../../model';
 import { ConnectionService } from '../../store/connection/connection.service';
 import { ConnectionConfigurationService } from '../common/configuration/configuration.service';
+import { ConfigService } from '@syndesis/ui/config.service';
+import { OnChanges } from '@angular/core/src/metadata/lifecycle_hooks';
 
 @Component({
   selector: 'syndesis-connection-detail-info',
   template: `
     <h1>
       <dl class="dl-horizontal">
         <dt>
-          <img src="../../../assets/icons/{{ connection.connectorId }}.connection.png" height="46" width="46">
+          <img src="{{ icon }}" height="46" width="46">
```

Comment:
Shouldn't this be `[src]="icon"`?

Should really change that height/width to be set via stylesheet too, though I see the original code didn't bother to either.